### PR TITLE
Added Traditional Chinese translation for Windows Installer

### PR DESCRIPTION
**Pull request by Jackmoo**
Added Traditional Chinese translation for Windows Installer

_added_
_STILL UNDER TESTING_
/dist/windows/languages/ChineseTraditional.isl

_modified_
/dist/windows/windows-installer.iss

### CODE REVIEW

File: dist/windows/windows-installer.iss [hunk 1]
```diff
@@ -63,6 +63,7 @@ Name: "ptbr"; MessagesFile: ".\languages\BrazilianPortuguese.isl"
 Name: "nl"; MessagesFile: ".\languages\Dutch.isl"
 Name: "ge"; MessagesFile: ".\languages\German.isl"
 Name: "pt"; MessagesFile: ".\languages\Portuguese.isl"
+Name: "zh-TW"; MessagesFile: ".\languages\Trad_Chinese.isl"
```
Comment (Jackmoo) [inline dist/windows/windows-installer.iss]: contributor used zh-tw not zh-TW in app/language/zh-tw.json
not sure if there's problem

**Comment by Patineta** [comment]
@Jackmoo Awesome! The InnoSetup guys would probably love this as well, so you should throw them a PR too. Regarding the .json files for translations, they are always in lowercase for compatibility.

If it's not too much to ask:
- Rename the file to Chinese.isl so it's consistent with the other languages.
- Just translating the lines we use for the installer is enough for now. We're releasing a new Beta in a few days, so the sooner we get this in the better.

**Comment by Jackmoo** [comment]
@Patineta Thanks! I've finished almost the whole translation in InnoSetup. However, since there's 2 types of Chinese, Traditional Chinese (used by HK, TW..etc) and Simplified Chinese(In mainland China), they used different LanguageID and CodePage(TW - BIG5, China - GB, not sure about HK). So It's better not to be confused with each other. 

Or maybe Chinese_Trad would be an option?

**Comment by Patineta** [comment]
@Jackmoo You can try TraditionalChinese.isl / SimplifiedChinese.isl, which is what we're doing with Portuguese and it seems to work well. Thanks!

**Comment by Patineta** [comment]
@Jackmoo Merged! Thanks a lot for this translation, it was a ton of work and we really appreciate it! You should try sending it to the InnoSetup guys, who'll be more than happy to merge it into their github as well.

**Comment by Jackmoo** [comment]
Thanks! I will contact them as well.

**Comment by Jackmoo** [comment]
@Patineta I'm so sorry. Due to the browser/website auto-transformation, the content in icl was transformed from Big5 to unicode (I used the website for commitment ). That may cause some problems. I will fix this ASAP.

**Pull request merged**